apps/ident-pool/nais.yml

```diff
@@ -35,5 +35,23 @@ spec:
     requests:
       cpu: 1000m
       memory: 4096Mi
+  accessPolicy:
+    inbound:
+      rules:
+        - application: testnav-pdl-forvalter
+          cluster: dev-gcp
+        - application: testnav-pdl-forvalter-dev
+          cluster: dev-gcp
+        - application: tps-forvalteren-dev
+          cluster: dev-fss
+        - application: tps-forvalteren
+          cluster: dev-fss
+        - application: app-1
+          namespace: plattformsikkerhet
+          cluster: dev-gcp
```

Comment:
Jeg sjekket og du hadde rett, testnorge-skd bruker også identpool så den må legges til i inbound rules :)

Reply:
OK, jeg legger til! 😄

Reply:
Men testnorge-skd sender ikke med noe access token for kall til ident-pool. Så hvis det kreves nå så må ident-pool consumer i testnorge-skd også oppdateres ;)

Reply:
Har lagt til sikkerhet i ident-pool for å kunne kalle tps-messaging-service, men foreløpig krever ikke ident-pool noe autentisering, så det går bra.